DataFormats/HGCalReco/interface/TICLCandidate.h

```diff
@@ -0,0 +1,68 @@
+#ifndef DataFormats_HGCalReco_TICLCandidate_h
+#define DataFormats_HGCalReco_TICLCandidate_h
+
+#include "DataFormats/Candidate/interface/LeafCandidate.h"
+#include "DataFormats/Common/interface/Ref.h"
+#include "DataFormats/HGCalReco/interface/Trackster.h"
+#include "DataFormats/Math/interface/Point3D.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+
+// A TICLCandidate is a lightweight physics object made from one or multiple Tracksters.
+
+namespace ticl {
```

Comment:
at some point, I thought we had agreed to use the `ticl` namespace only for classes that don't have `TICL` in the name

Reply:
Removed ticl namespace.